File: Java Programming/Competitive Programming/Coding Challenges 8/program5.java
// 5. Accept one number from user and print that number of * on screen.

class Program5
{
    static void Accept(int iNo)
    {
        int iCnt = 0;

        for (iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("*\t");
        }
    }

    public static void main(String[] args)
    {
        int iValue = 0;

        iValue = 5;

        Accept(iValue);
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 8/program5.cpp
//5.Write a program which accept number from user and return difference between summation of all its factors and non factors.
//Input : 12
//Output : -34 (16 - 50)
//Input : 10
//Output : -29 (8 - 37)

#include <iostream>
using namespace std;

int FactDiff(int iNo)
{
    int i = 1;
    int iFactSum = 0;
    int iNonFactSum = 0;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    while (i < iNo)
    {
        int iResult = iNo % i;

        if (iResult == 0)
        {
            iFactSum = iFactSum + i;
        }

        if (iResult != 0)
        {
            iNonFactSum = iNonFactSum + i;
        }

        i++;
    }

    return (iFactSum - iNonFactSum);
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    cout << "Enter number ";
    cin >> iValue;

    iRet = FactDiff(iValue);

    cout << iRet;

    return 0;
}